src/admin/lbps/dashboard/dashboard.html

```diff
@@ -28,38 +28,46 @@
             ${selectedLbp.metadata.general.projectName}
           </div>
           <div if.bind="selectedLbp.address">
-            <div class="featureContainer" if.bind="!selectedLbp.hasEnoughProjectTokens">
+            <div class="featureContainer" if.bind="selectedLbp.uninitialized">
               <div class="featureTitle">
-                Fund LBP
+                Approve tokens and Fund LBP
```

Comment:
Can you please make this a little cleaner, like:

Required to fund the LBP:

   [XXX] [funding token info] 
   [YYY] [project token info]

Reply:
okay, will do it.

Reply:
done, but need to check. For that I'll need to create a new LBP.